src/JsonSchema/Enum.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace JsonSchema;
+
+class Enum extends \MabeEnum\Enum
+{
```

Comment:
Good point - I assumed based on your summary, but you do mention extending `ConstraintError` explicitly. Let's leave this PR as you originally had it :-).